l10n_cn_sequence_voucher_manual/__init__.py

```diff
@@ -0,0 +1,2 @@
+# __author__ = cysnake4713@gmail.com
```

Comment:
@cysnake4713   Thanks for your contribution to the OCA. 

Please create one pull request per module (the other modules besides are already included in another PR.

Besides, please let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code..
In general, please follow the principles: 
* Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
* you need to make sure the code is PEP8 compliant for example and create a README file
* Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
* We recommend to include a screenshot as well to help new comers (nice to have)
* Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3
* Please sort all imports in alphabetical order following: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports